beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -430,6 +449,28 @@ impl<T: BeaconChainTypes> DataAvailabilityCheckerInner<T> {
         }
     }
 
+    /// Potentially trigger reconstruction if:
+    /// - Our custody requirement is all columns
+    /// - We >= 50% of columns, but not all columns
```

Comment:
Source code below is self-explanatory enough
```suggestion
    /// Returns true if reconstruction should start. Note that reconstruction is started only once.
    /// Currently only supernodes expecting all columns will attempt reconstruction.
```